addons/editor-theme3/addon.json

```diff
@@ -364,6 +372,34 @@
       "id": "forums",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Insertion marker style",
+      "id": "markerStyle",
+      "type": "select",
+      "default": "none",
+      "potentialValues": [
+        { "id": "none", "name": "Gray" },
+        { "id": "fill", "name": "Colored with gray outline" },
+        { "id": "stroke", "name": "Gray with colored outline" },
+        { "id": "both", "name": "Colored" }
+      ]
+    },
+    {
+      "name": "Insertion marker opacity",
+      "id": "fillOpacity",
+      "type": "integer",
+      "min": 0,
+      "max": 100,
+      "default": 20
+    },
+    {
+      "name": "Insertion marker outline opacity",
```

Comment:
This setting doesn't work if the style is "gray" because of the `addon.settings.get("markerStyle") !== "none"` check in the userscript.